server/controllers/api/index.js

```diff
@@ -13,4 +15,8 @@ router.get('/flights', flights);
 router.get('/flight/:flightId', updateFlight);
 router.post('/flight/:flightId', editFlightInformation);
 
+router.post('/check_flight', getFlightfromFlightNo);
+router.post('/get_flight_details', updateFlight);
```

Comment:
I think `router.get('flight/:flightId'`, `router.post('/check_flight'` and `router.post('/get_flight_details'` are all doing nearly the same thing - reading a flight.  You could probably combine them all into the `router.get('/flight/:flightId'` - that has the most REST-correct path, and is using `get` (since all of these cases are just reading the flight, not making changes to it).